apps/shipments/models/note.py

```diff
@@ -25,6 +25,8 @@ class ShipmentNote(models.Model):
     id = models.CharField(primary_key=True, default=random_id, max_length=36)
     message = models.TextField(validators=[MaxLengthValidator(500), MinLengthValidator(1)])
     user_id = models.CharField(null=False, max_length=36)
+    username = models.CharField(null=True, max_length=250)
```

Comment:
The Django username field has a max_length of 150

Reply:
Correct but then we append to this the organization name. example: `Clovis [ShipChain]`.
That is why I rounded it at 250 just in case.